### DISCUSSION TITLE
Barcode scanner auto populate field

### DISCUSSION THREAD

**Discussion by leyshanth** [Budibase/budibase#13533] [Help]
I need assistance in building an app to manage product expiry dates. I have two tables: one containing all product details and the other to capture expiry dates. What I envision is a form integrated with a barcode scanner. When I scan a product code, the app should automatically retrieve the corresponding details from the product table and populate the form. After reviewing and adding expiry date information, I should be able to save all the data to the second table. Could someone please provide a step-by-step guide on how to achieve this? Sorry, I'm a complete beginner.

**Comment by ATJonzie** [comment]
did you ever figure anything out?
